js/pages/geppetto/geppetto.ejs

```diff
@@ -45,7 +45,10 @@ $content
 
 
 <div id="mainContainer">
-	<div id="loadingText" style="text-align:center; font-family: consolas; margin-top: 50px;} ">LOADING... [~3MB]</div>
+	<div id="loadingText" style="display:grid; height: '100%'; text-align:center; font-family: consolas;">
+		<img src="geppetto/build/splash.png" style="max-width: 101%;max-height: 100; margin-top: -9px;margin-left: -8px;" alt="" onerror='document.getElementById("loadingMessage").style.display="block"' >
```

Comment:
@ddelpiano just two doubts:
- The negative margins and max-width 101% don't look very clean, seems something related to a specific image we re loading.
- Where the image `geppetto/build/splash.png` comes from? Do we have to place directly into the build directory or there something in webpack that is moving this file there?

Reply:
fixing this.